external/fv3fit/fv3fit/emulation/layers/architecture.py

```diff
@@ -372,6 +372,15 @@ def call(self, rnn_outputs: tf.Tensor) -> Mapping[str, tf.Tensor]:
         return fields
 
 
+_ARCHITECTURE_KEYS = (
```

Comment:
Centralize so it's easier to make sure the tests across all keys stay consistent.